src/main/java/br/com/excaladashboard/models/Funcionarios.java

```diff
@@ -0,0 +1,132 @@
+package br.com.excaladashboard.models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "funcionarios")
+public class Funcionarios {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "nome_completo", length = 150)
+    private String nomeCompleto;
+
+    @Column(name = "email_corporativo", length = 150)
+    private String emailCorporativo;
+
+    @Column(name = "email_facebook", length = 150)
+    private String emailFacebook;
+
+    @Column(name = "categoria", length = 256)
+    private String categoria;
+
+    @Column(name = "cpf_funcionario")
+    private String cpfFuncionario;
```

Comment:
Não precisa colocar funcionario no final porque ja sabemos que é um funcionário
```suggestion
    @Column(name = "cpf")
    private String cpf;
```